Makefile

```diff
@@ -275,6 +276,9 @@ bin/releases/git-lfs-%-$(VERSION).zip : $(RELEASE_INCLUDES) bin/git-lfs-%.exe
 	@mkdir -p bin/releases
 	zip -j -l $@ $^
 
+bin/releases/git-lfs-$(VERSION).tar.gz :
```

Comment:
Even though this is fairly self-describing, could you add a comment explaining what this recipe makes to be consistent with the surrounding recipes?

Reply:
Yeah, of course.